ArcGISRuntimeSDKQt_QMLSamples/Layers/QueryOGCAPICQLFilters/QueryOGCAPICQLFilters.qml

```diff
@@ -85,119 +97,118 @@ Rectangle {
-                            validator: RegExpValidator { regExp: /(0[1-9]|1[012])[-](0[1-9]|[12][0-9]|3[01])[-](19|20)\d\d/ }
-                            placeholderText: "MM-DD-YYYY"
-                        }
+                TextField {
+                    id: toDate
+                    anchors.verticalCenter: parent.verticalCenter
+                    width: 200
+                    text: ""
+                    selectByMouse: true
+                    validator: RegExpValidator {
+                        regExp: /(0[1-9]|1[012])[-](0[1-9]|[12][0-9]|3[01])[-](19|20)\d\d/
                     }
+                    placeholderText: "MM-DD-YYYY"
+                }
+            }
 
             Button {
                 anchors.horizontalCenter: parent.horizontalCenter
                 text: "Query"
 
-
                 TimeExtent {
                     id: time
                     startTime: new Date(fromDate.text)
                     endTime: new Date(toDate.text)
-                    }
-
+                }
 
                 onClicked: {
                     if (!ogcFeatureCollectionTable)
-                        return;
+                        return
 
                     // create the parameters
-                    const queryParams = ArcGISRuntimeEnvironment.createObject("QueryParameters",
-                                                                              {
-                                                                                  geometry: mapView.currentViewpointExtent.extent,
-                                                                                  whereClause: whereClauseMenu.currentText,
-                                                                                  maxFeatures: maxFeatureField.text,
-                                                                                  timeExtent: time
-                                                                              });
+                    const queryParams = ArcGISRuntimeEnvironment.createObject(
+                                          "QueryParameters", {
+                                              "geometry": mapView.currentViewpointExtent.extent,
+                                              "whereClause": whereClauseMenu.currentText,
+                                              "maxFeatures": maxFeatureField.text,
+                                              "timeExtent": time
+                                          })
 
                     // Populate the feature collection table with features that match the parameters,
                     // clear the cache to prepare for the new query results, and store all table fields
-                    ogcFeatureCollectionTable.populateFromService(queryParams, true, []);
+                    ogcFeatureCollectionTable.populateFromService(queryParams,
+                                                                  true, [])
```

Comment:
This seemed fine as it was. Any reason for the change? Also JS doesn't need them, but we include the semicolons regardless.
```suggestion
                    ogcFeatureCollectionTable.populateFromService(queryParams, true, []);
```